source/dub/internal/vibecompat/data/json.d

```diff
@@ -398,15 +398,15 @@ struct Json {
 	*/
 	inout(T) opCast(T)() inout { return get!T; }
 	/// ditto
-	@property inout(T) get(T)()
+	@property inout(T) get(T)(in string fileName = "DUMB_FILE_NAME")
```

Comment:
I'm not sure if `DUMB_FILE_NAME` is such a good name because:
- it's a valid filename, although a dumb one
- it does not give much information when it would suddenly appear in an error message

Additionally this should be a `static immutable string defaultJsonFileName` at the start of the file, to avoid writing this magic value out everywhere.

I think something like `(.json)` would be better.